package.json

```diff
@@ -17,16 +17,17 @@
     "build:node": "tsc -p ./tsconfig.prod.json",
     "build:browser": "tsc -p ./tsconfig.browser.json && npm run bundle && rm -rf dist.browser",
     "bundle": "webpack",
-    "client:start": "tsc -p tsconfig.prod.json && node dist/bin/cli.js",
+    "client:start": "ts-node bin/cli.ts",
```

Comment:
If I recall correctly I think I made the decision some time ago to use the JS sources here to have this closer to the consuming scenario since when people are installing the library they also can't use the `ts-node` invocation with the `TypeScript` sources. We likely have to think this through properly once anyhow though how we do the client invocation on package distribution before the next release. So if we use this `client:start` just for our own development purposes this should be ok respectively also beneficial to have the compilation skipped.

Reply:
Ah ok makes sense, yeah the new CLI tape test is using the built `dist` so it should be covered well with that, this `client:start` can just be our own developer convenience shortcut.